root/us-east-1/notifications/costs_tools_monitoring.tf

```diff
@@ -54,7 +105,7 @@ module "notify_slack_monitoring_costs" {
   # Creation Flags
   #
   create           = true
-  create_sns_topic = true
+  create_sns_topic = false
```

Comment:
@joseapeinado what happens if you are deploying the layer for the first time? I think this was set to true to create the SNS topic 

Can you validate this? Thanks!

Reply:
I've validated the creation of the layer from the scratch and it works nicely.